core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java

```diff
@@ -325,4 +307,138 @@ void findAllTest() {
 
         assertEquals(expectedList, placeService.findAll());
     }
-}
\ No newline at end of file
+
+    @Test
+    void updateTest() {
+        PlaceUpdateDto placeUpdateDto = new PlaceUpdateDto();
+        placeUpdateDto.setOpeningHoursList(openingHoursList);
+        placeUpdateDto.setDiscountValues(discountValuesDto);
+        placeUpdateDto.setName("new Name");
+        placeUpdateDto.setCategory(categoryDto);
+        when(categoryService.findByName(category.getName())).thenReturn(category);
+        when(placeRepo.findById(placeUpdateDto.getId())).thenReturn(Optional.of(place));
+        when(modelMapper.map(placeUpdateDto.getLocation(),Location.class)).thenReturn(location);
+
+        Place updatedPlace = placeService.update(placeUpdateDto);
+
+        assertEquals(placeUpdateDto.getName(), updatedPlace.getName());
+        assertEquals(placeUpdateDto.getCategory().getName(), updatedPlace.getCategory().getName());
+        verify(modelMapper).map(placeUpdateDto.getLocation(), Location.class);
+        verify(locationService).update(place.getLocation().getId(), location);
+    }
+
+    @Test
+    void deleteByIdTest() {
+        when(placeRepo.findById(place.getId())).thenReturn(Optional.of(place));
+        when(placeRepo.save(place)).thenReturn(place);
+
+        placeService.deleteById(place.getId());
+
+        assertEquals(PlaceStatus.DELETED, place.getStatus());
+        verify(notificationService, times(0)).sendImmediatelyReport(any());
```

Comment:
There is a more readable version 
`verify(notificationService, never()).sendImmediatelyReport(any());`

Reply:
Done!